### DISCUSSION TITLE
When does farming begin?

### DISCUSSION THREAD

**Discussion by Viking8** [ericaltendorf/plotman#645] [Q&A]
(I took the training wheels off right away. CLI only!)
I have chia and plotman setup and plots are being created, but I have no clue how to tell if chia is farming them now. Does plotman trigger farming as soon as the plot has moved to its final resting place, or does chia, or do I need to do something to get it started?

Thanks!

**Comment by mruckli** [comment]
Chia itself is responsible for the farming. Plotman only manages the creation of plots.
You have to have chia farmer running which points to the directory where your plots end up. Chia will periodically check those directories and once a new plot is added it will start farming.
This you then can see in the UI or cli with the right commands:
`chia farm summary`

**Reply by Viking8** [reply]
Thanks of the info. I have another question. What are the right commands?

**Reply by mruckli** [reply]
I posted an example above which should output a summary. From the response you should be able to deduce if your plots are being picked up.
Otherwise it depends on what you want to achieve in regards to what the correct commands are ;)

**Reply by Viking8** [reply]
I just did that and the output is as follows...

Exception from 'wallet' {'error' : "'NoneType' object has no attribute 'tx_store'",  'success' : False}
Farming Status: Syncing
Total chia farmed : Unknown
User Transaction Fees: Unknown
Block Rewards: Unknown
Last height farmed: Unknown
Plot Count: 5
etc. etc.
